RecoTracker/IterativeTracking/python/DetachedQuadStep_cff.py

```diff
@@ -441,6 +461,7 @@
                                 detachedQuadStepTracks*
                                 detachedQuadStepClassifier1*detachedQuadStepClassifier2*
                                 detachedQuadStep)
+trackingPhase1.toReplaceWith(DetachedQuadStep, DetachedQuadStep.copyAndExclude([detachedQuadStepHitTriplets]))
```

Comment:
You mean making the default in the file CA, and customize it to `PixelQuadrupletEDProducer` for the "previous phase1" and phase2? If yes, that kind of reorganization will be part of future cleanup PR (also for other quad steps).